source/postprocess/memory_statistics.cc

```diff
@@ -58,13 +58,12 @@ namespace aspect
 
       if (this->is_stokes_matrix_free())
         {
-          double mg_transfer_mem = this->get_stokes_matrix_free().get_mg_transfer_A().memory_consumption()
-                                   + this->get_stokes_matrix_free().get_mg_transfer_S().memory_consumption();
+          const double mg_transfer_mem = this->get_stokes_matrix_free().get_mg_transfer_A().memory_consumption()
```

Comment:
I think this should be a `std::size_type` or similar, not a `double`.

Reply:
We are storing all quantities in memory statistics as doubles and in MB. Maybe because you can see values less than 1 MB? If you want this changed, can we do this in a separate PR?

Reply:
I see. No, not necessary unless you really want!